src/style.css

```diff
@@ -1,3 +1,88 @@
+html {
+  /* stylelint-disable-next-line */
+  font-family:
+  Roboto, Arial,
+  Helvetica Neue,
+  Helvetica,
+  sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. The current font stack includes Roboto, Arial, Helvetica Neue, Helvetica, sans-serif which is good, but ensure that 'sans-serif' is indeed the fallback font you wish to use in case none of the others are available.